cmd/cadvisor.go

```diff
@@ -87,6 +87,7 @@ var (
 		container.ProcessSchedulerMetrics:        struct{}{},
 		container.ProcessMetrics:                 struct{}{},
 		container.HugetlbUsageMetrics:            struct{}{},
+		container.ReferencedMetric:               struct{}{},
```

Comment:
WDYT about `ReferencedMemoryMetrics` instead?  Just Referenced seems a little generic...

Reply:
and similarly `referenced_memory` for the flag string